src/main/scala/net/akmorrow13/endive/utils/Window.scala

```diff
@@ -82,6 +85,14 @@ case class Window(tf: TranscriptionFactors.Value,
   }
 }
 
+
+case class FeaturizedLabeledWindow(labeledWindow: LabeledWindow, features: DenseVector[Double]) {
+
```

Comment:
i just did a grep, we only use it in deprecated pipelines so this change won't break anything.